src/UI/Internal/Utils/Element.elm

```diff
@@ -33,6 +33,13 @@ title value =
         |> Element.htmlAttribute
 
 
+id : String -> Attribute msg
```

Comment:
Mental note: Maybe we would want to have  this in the future as an exposed function